Makefile

```diff
@@ -125,15 +125,15 @@ copy_terraform_files:
 	cp -r terraform_files/* build/terraform/;\
 
 run_terraform: copy_terraform_files
-	skipper make _run_terraform $(SKIPPER_PARAMS)
+	skipper make $(SKIPPER_PARAMS) _run_terraform
 
 _run_terraform:
 		cd build/terraform/ && \
 		terraform init -plugin-dir=/root/.terraform.d/plugins/ && \
 		terraform apply -auto-approve -input=false -state=terraform.tfstate -state-out=terraform.tfstate -var-file=terraform.tfvars.json
 
 destroy_terraform:
-	skipper make _destroy_terraform $(SKIPPER_PARAMS)
+	skipper make $(SKIPPER_PARAMS) _destroy_terraform
```

Comment:
nope,  it not right. For some reason skipper wants them after command

Reply:
Why do you think so?